container/ip-address-expansion/GUIsrc/presentation/constants.js

```diff
@@ -0,0 +1,32 @@
+import fields from 'json!./form_fields.json';
+
+const constants = {
+  timeout: 1000,
+  api: {
+    configuration: '/mgmt/shared/iapp/ipAddressExpansion',
+    updateDevice: '/mgmt/shared/iapp/ipAddressExpansion/devices',
+    updatePool: '/mgmt/shared/iapp/ipAddressExpansion/config',
+    tokenUrl: '/mgmt/shared/authn/login'
+  },
+  fields,
+  getBlockFullPath: id => `https://localhost/mgmt/shared/iapp/blocks/${id}`,
+  maxErrorCount: 10,
+  appState: {
+    ONLINE: 'ONLINE',
+    OFFLINE: 'OFFLINE',
+    DISABLED: 'DISABLED',
+    ERROR: 'ERROR',
+    TRANSITION: 'TRANSITION',
+    UNKNOWN: 'UNKNOWN',
+  },
+  blockState: {
```

Comment:
I think iapp-address-expansion is a iControl LX and not using block state machine.. Probably Alex copied this from other projects

Reply:
yep, cleaned it up a bit